ftplugin/ledger.vim

```diff
@@ -328,3 +348,11 @@ endf "}}}
 function! s:count_expression(text, expression) "{{{2
   return len(split(a:text, a:expression, 1))-1
 endf "}}}
+
+" Move the cursor to the specified column, filling the line with spaces if necessary.
+function! s:goto_col(pos) "{{{2
```

Comment:
I've put it in `ftplugin/ledger.vim` because other helper functions, also local to the script, are defined there (e.g., see `s:count_expression()` right there...). Should I move it anyway?